pkg/controller/account/generate.go

```diff
@@ -0,0 +1,38 @@
+package account
+
+import (
+	"fmt"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	"github.com/openshift/aws-account-operator/pkg/controller/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+
+// GenerateAccountCR returns new account CR struct
+func GenerateAccountCR(namespace string) *awsv1alpha1.Account {
+
+	uuid := utils.GenerateShortUID()
+
+	accountName := GenerateAccountCRName(uuid)
+
+	return &awsv1alpha1.Account{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      accountName,
+			Namespace: namespace,
+			Labels: utils.GenerateLabel(awsv1alpha1.IAMUserIDLabel, uuid),
```

Comment:
Wouldn't we want to label the Account CR and also suffix the IAM user with the full account name (emailID + uuid) instead of just the uuid?

That way if we start supporting multiple email IDs we're covered.

Reply:
The UID is the same as whats in the account name which is emailID + UID